althea_kernel_interface/src/netns.rs

```diff
@@ -7,7 +8,13 @@ impl dyn KernelInterface {
     /// that the lazy static for cross thread comms arch if a bit questionable by nature
     pub fn check_integration_test_netns(&self) -> u32 {
         if cfg!(feature = "integration_test") {
-            let ns = self.run_command("ip", &["netns", "identify"]).unwrap();
+            let mut ns = self.run_command("ip", &["netns", "identify"]);
+            while let Err(e) = ns {
+                warn!("Could not get netns name, retrying: {:?}", e);
+                sleep(std::time::Duration::from_secs(1));
```

Comment:
any chance this never exits the loop and stalls forever? are we keeping a time limit somewhere still

Reply:
found the timeout change nevermind